runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkStreamingTransformTranslators.java

```diff
@@ -912,12 +907,7 @@ public void translateNode(
       TypeInformation<WindowedValue<KV<K, OutputT>>> outputTypeInfo =
           context.getTypeInfo(context.getOutput(transform));
 
-      List<PCollectionView<?>> sideInputs;
-      try {
-        sideInputs = CombineTranslation.getSideInputs(context.getCurrentTransform());
-      } catch (IOException e) {
-        throw new RuntimeException(e);
-      }
+      List<PCollectionView<?>> sideInputs = new ArrayList<>();
```

Comment:
As above, please remove references to side inputs here. If it's not clear how to do that, please add a TODO and a new bug to track this removal.

Reply:
Done.